.github/workflows/builder.yaml

```diff
@@ -99,7 +99,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build ${{ matrix.addon }} add-on
-        if: steps.check.outputs.build_arch == 'true'
+        if: steps.check.outputs.build_arch == 'true' && steps.check.outputs.image != 'null'
```

Comment:
Move checking that to the check job, if image is null the output of build_arch should be false.